### DISCUSSION TITLE
第1节课笔记(6班)

### DISCUSSION THREAD

**Discussion by zhuyr97** [InternLM/Tutorial#64] [笔记]
第1节课笔记(6班)！无限进步！

**Comment by isLinXu** [comment]
Hertz(isLinxu):https://islinxu.github.io/llm-notes/InternLM/1-%E4%B9%A6%E7%94%9F%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E5%BC%80%E6%94%BE%E4%BD%93%E7%B3%BB.html

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/9684442f-3fa1-4663-a7df-4fd375ccb017](https://app.yinxiang.com/fx/9684442f-3fa1-4663-a7df-4fd375ccb017)
